MVC/system/classes/framework.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+  class framework
```

Comment:
Please add comments
Define the functions using @method
Define the parameter using @param
Define the return type using @param